### ISSUE TITLE
Error while running action

### ISSUE THREAD

**Issue by kislyuk** [marocchino/validate-dependabot#233]
`Error: request to https://dependabot-badges.githubapp.com/config_files/validate failed, reason: Hostname/IP does not match certificate's altnames: Host: api.dependabot.com. is not in the cert's altnames: DNS:*.gist.githubusercontent.com, DNS:gist.githubusercontent.com`

**Comment by marocchino** [comment]
Thank you for the report.
They must have noticed about stealing traffic.
It is easy to imitate IP and address and fix it, but it would be better to implement it in a different way. Let's take a look to see if there are any reusable parts of the code they've released.

**Comment by Sleepful** [comment]
Subscribing, was trying to add this today and sad face

**Comment by mark-stopka** [comment]
Hi folks any update / workaround on this?

**Comment by marocchino** [comment]
TL;DR sorry but, there is not clear answer here yet.

after research, It looks clear they use json schema but, there dose not opensource that schema.
 
I found [the json schema](
https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/dependabot-2.0.json) but I am not sure it is officially maintained one.

**Comment by eirnym** [comment]
I've saw Jetbrains products use similar schema

**Comment by mark-stopka** [comment]
Issue is logged with upstream https://github.com/dependabot/dependabot-core/issues/4605

**Comment by mark-stopka** [comment]
Following worked:

```
mark_stopka@mark-asus-zenbook-s:~> ajv validate --strict=log -s dependabot-2.0.json -d dependabot.yml && echo ok
strict mode: unknown keyword: "x-intellij-enum-metadata"
strict mode: missing type "number" for keyword "minimum" at "http://json-schema.org/draft-04/schema#/properties/assignees" (strictTypes)
strict mode: unknown keyword: "x-intellij-enum-metadata"
strict mode: missing type "number" for keyword "minimum" at "http://json-schema.org/draft-04/schema#/properties/reviewers" (strictTypes)
strict mode: unknown keyword: "x-intellij-enum-metadata"
schema dependabot-2.0.json is invalid
error: schema is invalid: data must have required property 'version'
ajv migrate -s dependabot-2.0.json -o new-dependabot-2.0.json
mark_stopka@mark-asus-zenbook-s:~> ajv validate --strict=log -s new-dependabot-2.0.json -d dependabot.yml && echo ok
strict mode: unknown keyword: "x-intellij-enum-metadata"
strict mode: missing type "number" for keyword "minimum" at "http://json-schema.org/draft-04/schema#/properties/assignees" (strictTypes)
strict mode: unknown keyword: "x-intellij-enum-metadata"
strict mode: missing type "number" for keyword "minimum" at "http://json-schema.org/draft-04/schema#/properties/reviewers" (strictTypes)
strict mode: unknown keyword: "x-intellij-enum-metadata"
dependabot.yml valid
ok
mark_stopka@mark-asus-zenbook-s:~>

**Comment by marocchino** [comment]
I reimplement this with schema-json and tag as v2.
